DigitalLearningSolutions.Web/Views/ApplicationSelector/Index.cshtml

```diff
@@ -0,0 +1,93 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.ApplicationSelector
+@using Microsoft.Extensions.Configuration
+
+@inject IConfiguration Configuration
+
+@model ApplicationSelectorViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/myAccount/myAccount.css")" asp-append-version="true">

```

Comment:
If we're using this css file here, maybe we need to consider either renaming it or splitting out the stuff that isn't exclusive to the my account page?

Reply:
We aren't actually using it, I think it was just retained in a copy/paste by mistake. The CSS is all NHS classes.